src/main/java/reposense/util/StringsUtil.java

```diff
@@ -31,10 +31,33 @@ public static String replaceSpecialSymbols(String regexString, String replacemen
         return SPECIAL_SYMBOLS.matcher(regexString).replaceAll(replacementCharacter);
     }
 
-    public static String addQuote(String original) {
+    /**
+     * Adds quotes to the string.
+     */
+    public static String addQuotes(String original) {
         return "\"" + original + "\"";
     }
 
+    /**
+     * Adds the appropriate quotation marks for a file path depending on the OS.
+     */
+    public static String addQuotesForFilePath(String filePath) {
+        if (SystemUtil.isWindows()) {
+            return "\"" + filePath + "\"";
```

Comment:
Is it possible to cover this line in the test as well?

Reply:
Unfortunately, I don't think there is a good way to test the escaping specifically. We can add a test case just to check if quotation marks are added which does not seem very useful. For this case, it was to avoid mishandling of spaces in the repo directory and the test cases are located [here](https://github.com/reposense/RepoSense/commit/2c05624a7c59e3d47c7a5ddf97f800b8655b7b99#diff-e940f5fe270077934d6b1a6ecb957d627681a5a38abaea308070d46b27e130ebR36). 

For testing the escaping specifically, I don't think there's a good way to test this for Windows filesystems. The CMD's `echo` command echoes the quotation marks compared to bash's `echo` command which strips the quotation marks and does symbol substitution where appropriate.

Reply:
I think it's more that there isn't a specific test case that runs this line of code at all since the test case I added only runs on non-Windows systems.